src/main/resources/assets/lambdabettergrass/lang/ru_ru.json

```
@@ -0,0 +1,12 @@
+{
+  "lambdabettergrass.menu.info.1": "Мод использует ресурспак, чтобы определить, какие блоки затронуты.",
+  "lambdabettergrass.menu.info.2": "Включение или отключение функции улучшенного дёрна/снега на блоках",
```

```suggestion
  "lambdabettergrass.menu.info.2": "Это значит, включение/отключение влияния улучшенной травы/снега на блоки",
```
Более близко к оригинальному контексту